embassy-stm32f4/src/serial.rs

```diff
@@ -181,6 +232,8 @@ where
                 () = fut_idle.fuse() => {},
             }
 
+            let bytes = RSTREAM::get_number_of_transfers() as usize;
```

Comment:
This probably gives the remaining number of transfers, not the number of transfers that were completed.

Reply:
I think it's correct, for this reason:  the registers read by this function appear to be the same ones as that read by examples written in C.

Reply:
From the manual:

> Number of data items to be transferred (0 up to 65535). This register can be written only
when the stream is disabled. When the stream is enabled, this register is read-only,
**indicating the remaining data items to be transmitted**. This register decrements after each
DMA transfer.
Once the transfer has completed, this register can either stay at **zero** (when the stream is in
normal mode) or be reloaded automatically with the previously programmed value in the
following cases:
>
>– when the stream is configured in Circular mode.
>– when the stream is enabled again by setting EN bit to '1'
>
>If the value of this register is zero, no transaction can be served even if the stream is enabled.